homeassistant/components/feedreader.py

```diff
@@ -103,8 +120,51 @@ def _publish_new_entries(self):
         self._firstrun = False
 
 
+class StoredData(object):
+    """Abstraction over pickle data storage."""
+
+    def __init__(self, data_file):
+        """Initialize pickle data storage."""
+        self._data_file = data_file
+        self.lock = Lock()
+
+    def _get_data(self):
+        """Fetch data stored into pickle file."""
+        if exists(self._data_file):
```

Comment:
OK, I thought of it but felt a bit overkill for the use case (one feed update every hour, maximum 20 read/writes per feed), but why not, will do